DS and Algorithm/SeleniumTest/SeleniumExample.java

```diff
@@ -0,0 +1,38 @@
+// src/main/java/SeleniumExample.java
+public class SeleniumExample {
+    public static void main(String[] args) {
+        // Initialize WebDriver
+        WebDriver driver = WebDriverSetup.initializeChromeDriver();
+
+        // Navigate to the desired website
+        driver.get("https://www.google.com");
+
+        // Perform a search using WebInteraction class
+        WebInteraction.performSearch(driver, "Selenium WebDriver");
+
+        // Wait for a few seconds to see the search results
+        try {
```

Comment:
**Suggestion**: Replace Thread.sleep with WebDriverWait to create more reliable and efficient waits. Thread.sleep can lead to either unnecessary waiting or flaky tests if the wait time is not sufficient. <br> **Code Suggestion**: 
 ```
 - Thread.sleep(3000);
 + new WebDriverWait(driver, Duration.ofSeconds(3)).until(webDriver -> ((JavascriptExecutor) webDriver).executeScript("return document.readyState").equals("complete"));
 ```